RBDM/Integer.h

```diff
@@ -0,0 +1,20 @@
+#ifndef Integer_h_
+#define Integer_h_
+
+#include"Base.h"
+
+class Integer : public BaseForType
+{
+private:
+	long value;
```

Comment:
Why do we use long here?
I suggest use int
Or, if you want to extend specter of values that may be contained here, use long long instead of long.